lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -87,6 +87,27 @@ class AwsCompileEventBridgeEvents {
           required: ['inputTemplate'],
           additionalProperties: false,
         },
+        retryPolicy: {
+          type: 'object',
+          properties: {
+            maximumEventAge: {
+              type: 'integer',
+              minLength: 60,
```

Comment:
This is integer type and `minLength` and `maxLength` is e.g. for `string` type. I believe you wanted to use `maximum`, etc?